_data/simple-icons.json

```diff
@@ -2030,6 +2030,11 @@
             "hex": "000020",
             "source": "http://expo.io/brand/"
         },
+        {
+            "title": "Express",
+            "hex": "000000",
+            "source": "https://github.com/openjs-foundation/artwork/tree/master/projects/express"
```

Comment:
Could you link to the specific version of this as the source?
```
https://github.com/openjs-foundation/artwork/blob/ac43961d1157f973c54f210cf5e0c9c45e3d3f10/projects/express/express-icon-black.svg
```

Reply:
@adamrusted Done